app/components/CounterButton/CounterButton.Styled.jsx

```diff
@@ -9,13 +9,22 @@ const handleColorType = (props) => {
   return '#31425a';
 };
 
+const setCursorType = (props) => {
+  if (props.processingFormSubmission) {
+    return 'not-allowed;';
+  }
+  else {
```

Comment:
This else is not necessary. Just return your string.

Reply:
You are totally right, sorry about that, I've removed this redundant else clause, thank you!